server/api/api/endpoints/projects_v2.py

```diff
@@ -179,7 +185,20 @@ async def _delete_project(
     project_name: str,
     deletion_strategy: mlrun.common.schemas.DeletionStrategy,
     auth_info: mlrun.common.schemas.AuthInfo,
+    wait_for_project_deletion: bool,
 ):
+    def _verify_project_is_deleted():
+        try:
+            get_project_member().get_project(
```

Comment:
this should wait for the mlrun background job that was triggered in the v1 endpoint (not yet implemented)
but since you don't have the task name you can query the project in mlrun DB